File: KantechGatewayApp/Jobs/VendorDailyProcessor.vb
Namespace KantechGatewayApp.Jobs
    Public Class VendorDailyProcessor
        Inherits BaseProcessor
        Protected Overrides Sub HandleRow(row As Dictionary(Of String, String))
            Infrastructure.Logger.Info($"[VendorDaily] emp={row.GetValueOrDefault("employee ID")} card={row.GetValueOrDefault("Card ID")} -> daily access set", _logSub)
        End Sub
    End Class
End Namespace

File: KantechGatewayApp/Jobs/VendorDailyProcessor.cs
using System.Collections.Generic;

namespace KantechGatewayApp.KantechGatewayApp.Jobs
{
    public class VendorDailyProcessor : BaseProcessor
    {
        protected override void HandleRow(Dictionary<string, string> row)
        {
            Infrastructure.KantechJobApi.TriggerJobApi(_jobKey, row);
            Infrastructure.Logger.Info($"[VendorDaily] emp={row.GetValueOrDefault("employee ID")} card={row.GetValueOrDefault("Card ID")} -> daily access set", _logSub);
        }
    }
}